### DISCUSSION TITLE
To add ability separate out tests and profiles

### DISCUSSION THREAD

**Discussion by fkhamdamov** [yaalalabs/fixyl#14] [Ideas]
Do you think it would be possible to add a functionality which would allow to categorize tests, scenarios based on profiles to which it relates to?

For example: I have set of profiles which I use to run tests only on particular test environments (env1) and another set which i use to run test in another environment. Now all of them are under one tab and i have huge list of profiles. Would be great to separate them somehow. Also to have set of scenarios or favorited messages to be associated to the profile on which it was created and not total library which can't be used if accidentally logged by another profile.

**Comment by DhananjayaR** [comment]
Got your point. Need to clarify a certain aspect of the current implementation of the tool. What you see in the scenarios and favorite section depends on the dictionary (specifically the name) you've used when creating the profiles. If you use the same dictionary file (same name) the favorites and scarious will be shared across those profiles since they have the same dictionary. As a workaround, if you can change the dictionary name to two distinct names, the favorites and scarious will not be visible to one another.

**Comment by DhananjayaR** [comment]
Btw, apologies for the delayed response 🙂
